### DISCUSSION TITLE
Pop VSCode dialog box for warnings, errors etc

### DISCUSSION THREAD

**Discussion by KohlerVG** [glenn2223/vscode-live-sass-compiler#243] [Ideas]
The Live SASS Compiler errors only appear in the terminal, it would be nice have the errors show in a dialog box instead, i think there is a VSCode dialog function you can use to do this...

**Comment by glenn2223** [comment]
It's a nice idea but the alert box on VSCode is very restrictive, as shown below. It doesn't respect linebreaks and you can't get it to open fully so it'll always pop up in the collapsed state.

**Collapsed** (no way to change it)
![image](https://user-images.githubusercontent.com/33450392/208152656-db9a3ed6-8b9f-4f77-8a57-68d80981cd87.png)

**Expanded** (newlines aren't respected)
![image](https://user-images.githubusercontent.com/33450392/208152707-6dfdc79e-117f-4003-b697-1c27633f92b0.png)
